### ISSUE TITLE
Modal: Links are not understandable - 1.4.1 Use of Color

### ISSUE THREAD

**Issue by syarwick23** [stephenravenscraft/loyola-template-4.0#119]
The links to open the modals only use colors to distinguish themselves as links . There is a font change but the text color does not have a sufficient contrast difference between it and normal unlinked text
